### PR TITLE
[4.0] Switcher ui

### PR DESCRIPTION
**Pull request by angieradtke**
### Summary of Changes

Changes of CSS for better UX - make options always visible for a better understanding

### Testing Instructions

go to article edit and take look  at the featured options

### CODE REVIEW

File: build/media_source/system/scss/fields/switcher.scss [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
+  border-radius: 3px;
 }
-
 .switcher input {
-  position: absolute;
-  top: 0;
-  z-index: 2;
-  opacity: 0;
+  width: 100px;
+  /* visibility: hidden;
```
Comment (brianteeman) [inline build/media_source/system/scss/fields/switcher.scss:15]: there should not be any code that is commented out

**Comment by brianteeman** [comment]
@rdeutz your attempt to remove _variables.css from this PR has instead deleted it from the branch

**Comment by rdeutz** [comment]
@brianteeman I am already looking into this to fix my mistake

**Comment by brianteeman** [comment]
I am not in favour of this change. It makes the screen much busier than it needs to be and it makes it harder to scan a page with a lot of options (sadly we have a lot of those pages)

**Comment by rdeutz** [comment]
closing this one here and making a new one because I messed it up

**Comment by angieradtke** [comment]
oh I see the code in the global- config etc is different - so there it isn't working-  I have to made some adaptions

**Comment by brianteeman** [comment]
@rdeutz Don't bother to remake it - we already have code for this

image

```
		<field
			name="featured"
			type="radio"
			label="JFEATURED"
			layout="joomla.form.field.radio.buttons"
			default="0"
			class="btn-group btn-group-yesno"
			>
			<option value="0">JNO</option>
			<option value="1">JYES</option>
		</field>
```'

**Comment by Bakual** [comment]
> layout="joomla.form.field.radio.buttons"

You actually can even remove that line, because this is the default layout anyway.

Just for reference: I've suggested getting rid of the switcher and using the btn-groups again back in 2017 with https://github.com/joomla/joomla-cms/pull/15946. You can read the comments there
I can always update and reactivate it if there is some interest.

**Comment by Bakual** [comment]
I redid the "Revert Switcher" PR so you can have a look 😄 
https://github.com/joomla/joomla-cms/pull/27952

**Pull request closed without merging**